packages/atomic/src/components/search/atomic-sort-expression/atomic-sort-expression.spec.ts

```diff
@@ -0,0 +1,219 @@
+import {html} from 'lit';
+import {beforeEach, describe, expect, it, type MockInstance, vi} from 'vitest';
+import {fixture} from '@/vitest-utils/testing-helpers/fixture';
+import type {AtomicSortExpression} from './atomic-sort-expression';
+import './atomic-sort-expression';
+
+describe('atomic-sort-expression', () => {
+  const renderSortExpression = async ({
+    label = 'Test Label',
+    expression = 'relevancy',
+    tabsIncluded = [],
+    tabsExcluded = [],
+    insideDropdown = true,
+  }: {
+    label?: string;
+    expression?: string;
+    tabsIncluded?: string[];
+    tabsExcluded?: string[];
+    insideDropdown?: boolean;
+  } = {}) => {
+    const template = html`<atomic-sort-expression
+      label="${label}"
+      expression="${expression}"
+      tabs-included="${JSON.stringify(tabsIncluded)}"
+      tabs-excluded="${JSON.stringify(tabsExcluded)}"
+    ></atomic-sort-expression>`;
+
+    const element = insideDropdown
+      ? await fixture<AtomicSortExpression>(html`
+          <atomic-sort-dropdown>${template}</atomic-sort-dropdown>
+        `)
+      : await fixture<AtomicSortExpression>(template);
+
+    return {
+      element: insideDropdown
+        ? (element.querySelector(
+            'atomic-sort-expression'
+          ) as AtomicSortExpression)
+        : element,
+    };
+  };
+
+  describe('when rendering with valid props', () => {
```

Comment:
We should also test that it sets the error if the label is missing/empty or if the expression is missing/empty (prop validation tests).